usecases/scaler/scaler.go

```diff
@@ -71,7 +71,7 @@ type BackUpper interface {
 // cluster is used by the scaler to query cluster
 type cluster interface {
 	// AllNames returns list of existing node in the cluster
```

Comment:
Would be good also to change the comment above instead of `AllNames` should be `Candidates`

Reply:
Done 👍